### PR TITLE
Unify tenant

### PR DESCRIPTION
**Pull request by ad-m**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -84,13 +84,13 @@ Dane uwierzytelniające zostały zapisane na platformie Linux / macOS w ``$HOME/
...
-Jeżeli wykorzystujesz więcej niż 1 tenant musisz dokonać wyboru aktywnie używanego poprzez pobranie identyfikatora właściwego::
+Jeżeli wykorzystujesz więcej niż 1 project musisz dokonać wyboru aktywnie używanego poprzez pobranie identyfikatora właściwego::
 
-    $ h1 tenant list --output table
+    $ h1 project list --output table
 
 Następnie zatwierdzenia tego wyboru::
 
-    $ h1 tenant select {{tenant_id}}
+    $ h1 project select {{project_id}}
```
Comment (fredericosilva) [inline README.rst]: lets remove _id and let the user use project name here

File: bin/generic/access/grant.js [hunk 2]
```diff
@@ -1,10 +1,11 @@
 'use strict';
 
 const Cli = require('structured-cli');
+const genericDefaults = require('bin/generic/defaults');
 
 const options = {
-    'tenant-id': {
-        description: 'Tenant Id',
+    'project-id': {
```
Comment (fredericosilva) [inline bin/generic/access/grant.js]: change project-id to project

File: bin/generic/access/grant.js [hunk 3]
```diff
@@ -1,10 +1,11 @@
...
+        description: 'Project Id',
```
Comment (fredericosilva) [inline bin/generic/access/grant.js]: change project-id to project

File: bin/generic/access/grant.js [hunk 4]
```diff
@@ -38,8 +34,8 @@ module.exports = function(resource) {
 function handleAccessGrant(resource) {
     return function(args) {
         return args.helpers.api.post(`${resource.name}/${args.id}/accessrights`, {
-            identity: args['tenant-id']
+            identity: args['project-id']
```
Comment (fredericosilva) [inline bin/generic/access/grant.js]: change project-id to project

File: bin/network/ip/create.js [hunk 5]
```diff
@@ -1,15 +1,11 @@
...
 
 module.exports = resource => Cli.createCommand('create', {
     description: 'IP create'
-  , plugins: [
-        require('bin/_plugins/loginRequired')
-      , require('bin/_plugins/tenantRequired')
-      , require('bin/_plugins/outputFormat')
-      , require('bin/_plugins/api')
-    ]
+  , plugins: genericDefaults.plugins
```
Comment (fredericosilva) [inline bin/network/ip/create.js]: use resource.plugins here

File: bin/project/access/grant.js [hunk 6]
```diff
@@ -18,12 +19,7 @@ const options = {
 
 module.exports = resource => Cli.createCommand('grant', {
     description: `Grant access rights for ${resource.name.toUpperCase()}`,
-    plugins: [
-        require('bin/_plugins/loginRequired'),
-        require('bin/_plugins/tenantRequired'),
-        require('bin/_plugins/outputFormat'),
-        require('bin/_plugins/api')
-    ],
+    plugins: genericDefaults.plugins,
```
Comment (fredericosilva) [inline bin/project/access/grant.js]: use resource.plugins here

File: bin/project/access/revoke.js [hunk 7]
```diff
@@ -12,16 +13,10 @@ const options = {
 
 module.exports = resource => Cli.createCommand('revoke', {
     description: `Revoke access rights for ${resource.name.toUpperCase()}`,
-    plugins: [
-        require('bin/_plugins/loginRequired'),
-        require('bin/_plugins/tenantRequired'),
-        require('bin/_plugins/outputFormat'),
-        require('bin/_plugins/api')
-    ],
+    plugins: genericDefaults.plugins,
```
Comment (fredericosilva) [inline bin/project/access/revoke.js]: use resource.plugins here

File: bin/project/access/revoke.js [hunk 8]
```diff
@@ -12,16 +13,10 @@ const options = {
...
     params: resource.params,
     options: options,
     handler: args => args.helpers.api
-        .delete(`${resource.url(args)}/accessrights/${args['tenant-id']}`)
+        .delete(`${resource.url(args)}/accessrights/s${args['identity']}`)
```
Comment (fredericosilva) [inline bin/project/access/revoke.js]: args.identity is more readable here

**Comment by ad-m** [comment]
All requested corrections applied.

**Pull request merged**